ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -21,6 +21,30 @@ const (
 	FailedToUpdateService            ServiceStatus = "FailedToUpdateService"
 )
 
+// These statuses should match Ray Serve's application statuses
+var ApplicationStatus = struct {
```

Comment:
nit: The naming `ApplicationStatus` here is a little tricky since it's the same with the field of `RayServiceStatus. ApplicationStatus` but they are different struct, so it would be a little bit confusing

Reply:
Ah, good catch! I renamed `ApplicationStatus` to `ApplicationStatusEnum` and `DeploymentStatus` to `DeploymentStatusEnum`.